src/Pipeline.php

```diff
@@ -15,12 +15,24 @@ class Pipeline implements PipelineInterface
      */
     private $processor;
 
+    /**
+     * Pipeline constructor.
+     *
+     * @param \League\Pipeline\ProcessorInterface|null $processor
+     * @param callable ...$stages
+     */
     public function __construct(ProcessorInterface $processor = null, callable ...$stages)
     {
         $this->processor = $processor ?? new FingersCrossedProcessor;
         $this->stages = $stages;
     }
 
+    /**
+     * One pipe for your task.
+     *
+     * @param callable $stage
+     * @return \League\Pipeline\PipelineInterface|$this
+     */
```

Comment:
Not needed, `PipelineInterface` already provides all the docblocks. Same for other docblocks added in this PR.

Reply:
For others -- ok. But for this docblock for `PipelineInterface` returns only interface without `$this`. IDE autocompletion works only within interface scope and not touch `process` method which should be there to be.